bolt-design-system/bolt
# Dev tool: IE11-only SCSS mixin

## Jira

N/A

## Summary

Adds a SCSS mixin for Bolt devs to define IE11 CSS.

## Details

Created `bolt-ie11-only` mixin under tools in the core. This will make it easier every time we need to write CSS that only applies to IE11.

## How to test

1. Run the branch locally.
2. Add the following test code to any component CSS to target that component:
```scss
@include bolt-ie11-only {
  * {
    background-color: red !important;
  }
}
```
3. Make sure whatever you targeted turned to red.
